modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/ColocatedSortAggregatePlannerTest.java

```diff
@@ -293,47 +295,17 @@ public void noSortAppendingWithCorrectCollation() throws Exception {
      */
     @Test
     public void emptyCollationPassThroughLimit() throws Exception {
-        assertPlan(TestCase.CASE_17,
-                hasChildThat(isInstanceOf(IgniteCorrelatedNestedLoopJoin.class)
-                        .and(input(1, isInstanceOf(IgniteColocatedSortAggregate.class)
-                                .and(input(isInstanceOf(IgniteLimit.class)
-                                        .and(input(isInstanceOf(IgniteSort.class)
-                                                .and(input(isTableScan("TEST")))
-                                        ))
-                                ))
-                        ))
-                ),
-                disableRules
-        );
+        RuntimeException e = assertThrows(RuntimeException.class,
+                () -> assertPlan(TestCase.CASE_17, isInstanceOf(IgniteRel.class), disableRules));
+        assertThat(e.getMessage(), containsString("There are not enough rules to produce a node with desired properties"));
```

Comment:
Maybe, it would be better to disable tests instead of rewriting their code?

Reply:
I thought about it, but decided to go with changing the test. First, I want to make sure such plan is impossible, so we need such test anyway. Second, this is not just side effect of my change, but rather intended result. When (or most probably "if") we fix problem with correlated sorted exchange, there is no guarantee the resulting plan will be the same.